op-guide/backup-restore.md

```diff
@@ -0,0 +1,116 @@
+category: advanced
+---
+
+# Backup and Restore
+
+## About
+
+This document describes how to back up and restore the data of TiDB. Currently, this document only covers full backup and restoration.
+
+Here we assume that the TiDB service information is as follows:
+
+|Name|Address|Port|User|Password|
+|:----:|:-------:|:----:|:----:|:------:|
+|TiDB|127.0.0.1|4000|root|*|
+
+In the process of backing up and restoring data, use the following tools:
+
+- `mydumper`: to export data from TiDB
+- `loader`: to import data into TiDB
+
+### Download TiDB Toolset (Linux)
+
+```bash
+# Download the tool package.
+wget http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz
+wget http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.sha256
+
+# Check the file integrity. If the result is OK, the file is correct.
+sha256sum -c tidb-enterprise-tools-latest-linux-amd64.sha256
+
+# Extract the package.
+tar -xzf tidb-enterprise-tools-latest-linux-amd64.tar.gz
+cd tidb-enterprise-tools-latest-linux-amd64
+```
+
+## Full Backup and Restoration Using `mydumper`/`loader`
+
+The `mydumper` is a powerful data backup tool. See more about [`mydumper`](https://github.com/maxbube/mydumper). 
+
+To back up data, use `mydumper` to export data from TiDB. To restore data, use `loader` to import data into TiDB.
+
+**Note:** Although TiDB supports backing up and restoring data using the official `mysqldump` tool of MySQL, its performance is worse than `mydumper`/`loader`. It will take up much time to back up and restore a large amount of data. Therefore, the tool `mysqldump` is not recommended.
+
+### Best Practices of Full Backup and Restoration Using `mydumper`/`loader` 
+
+To quickly back up and restore data (especially large amounts of data), refer to the following recommendations:
+
+- Keep the size of the data file exported using `mydumper` as small as possible. It is recommended to keep it within 64M. You can set the parameter `-F` to 64.
+- You can adjust the `loader` parameter `-t` based on the number of TiKV instances and the load. For example, you can set the parameter to `3 *(1 ～ n)` in the case of 3 TiKV scenarios. When the TiKV load is too high, and a large number of `backoffer.maxSleep 15000ms is exceeded` show in `loader` and TiDB log, adjust it to a smaller value appropriately. When the TiKV load is not too high, adjust it to a higher value appropriately.
+
+#### An Example of Restoring Data and Related Configuration 
+
+- The total amount of data exported using `mydumper` is 214G. Single table 8 columns, 2 billion rows of data.
+- Cluster topology.
+  - TiKV * 12
+  - TIDB * 4
+  - PD * 3
+- Set the `mydumper` parameter `-F` to 16, and `loader` parameter `-t` to 64.
+
+Result: It takes about 11 hours to import, with 19.4G/hour.
+
+### Back up Data from TiDB
+
+Use `mydumper` to back up data from TiDB.
+
+```bash
+./bin/mydumper -h 127.0.0.1 -P 4000 -u root -t 16 -F 64 -B test -T t1,t2 --skip-tz-utc -o ./var/test
+```
+
```

Comment:
Change the description of the command to the following:
In this command,

+ `-B test`: means the data is exported from the `test` database.
+ `-T t1,t2`: means only the `t1` and `t2` tables are exported.
+ `-t 16`: means 16 threads are used to export the data.
+ `-F 64`: means a table is partitioned into chunks and one chunk is 64MB.
+ `--skip-tz-utc`: the purpose of adding this parameter is to ignore the inconsistency of time zone setting between MySQL and the data exporting machine and to disable automatic conversion.